tests/testutils/artifactshare.py

```diff
@@ -67,7 +57,17 @@ def run(self, q):
             q.put(port)
 
             # Sleep until termination by signal
-            signal.pause()
+            signal.sigwait([signal.SIGTERM, signal.SIGINT])
+
+            server.stop(0)
+
+        # Save collected coverage data
+        try:
+            from pytest_cov.embed import cleanup
+        except ImportError:
+            pass
+        else:
+            cleanup()
```

Comment:
```suggestion
        with suppress(ImportError):
            from pytest_cov.embed import cleanup
           cleanup()
```
would that be more readable?

Reply:
It would be a bit more concise, however, pylint raises an 'import-error', which can be suppressed but reduces the readability advantage, in my opinion. Also, ignoring potential import errors inside the `cleanup()` function is not ideal, even though this is unlikely to be an issue. I'll leave it as is for now.